internal/contentutil/reference.go

```diff
@@ -15,10 +15,18 @@ limitations under the License.
 
 package contentutil
 
-import "github.com/opencontainers/go-digest"
+import (
+	"github.com/opencontainers/go-digest"
+	"oras.land/oras-go/v2/registry"
+)
 
 // IsDigest checks if the given string is a valid digest.
 func IsDigest(tagOrDigest string) bool {
 	_, err := digest.Parse(tagOrDigest)
 	return err == nil
 }
+
+// IsValidTag checks if the given string is a valid tag.
+func IsValidTag(tag string) bool {
+	return registry.Reference{Reference: tag}.ValidateReferenceAsDigest() != nil
+}
```

Comment:
Do you mean
```suggestion
func IsTag(tagOrDigest string) bool {
	return registry.Reference{Reference: tagOrDigest}.ValidateReferenceAsTag() == nil
}
```

Reply:
See: https://github.com/oras-project/oras/pull/1475#discussion_r1732233944

Reply:
Removed this part of code.